### PR TITLE
test: add test cases in ContractsByCode

### PR DESCRIPTION
**Pull request by 170210**
## Description
Add test cases in function ContractsByCode because we have not tested it.
This PR adds 8 test cases to this function, including #65 and #66. 

## How has this been tested?
Unit tests

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/wasmd/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/wasmd/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes. (not need)
- [ ] I have updated the documentation accordingly. (not need)
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml` (not need)

### CODE REVIEW

**Review by kokeshiM0chi — DISMISSED** [review]

**Comment by kokeshiM0chi** [comment]
I think it's okay to delete unnecessary items in the explanation of the PR template.

**Review by da1suk8 — DISMISSED** [review]

File: x/wasm/keeper/querier_test.go [hunk 1]
```diff
@@ -284,6 +284,127 @@ func TestQueryRawContractState(t *testing.T) {
...
+		adminAddr := contract.CreatorAddr
+		label := "demo contract to query"
+		contractAddr, _, err := keepers.ContractKeeper.Instantiate(ctx, contract.CodeID, contract.CreatorAddr, adminAddr, initMsgBz, label, initialAmount)
+		fmt.Println(contract.CodeID)
+		require.NoError(t, err)
+		return contractAddr
+	}
+
+	// related link: https://github.com/CosmWasm/wasmd/issues/1559
+	// When not set with a block, the contract will return results based on the order of address rather than instantiation (timestamp).
```
Comment (da1suk8) [inline x/wasm/keeper/querier_test.go:315]: > based on the order of address

What does this mean?

Comment (170210) [reply x/wasm/keeper/querier_test.go:315]: It means that the order of contract address

Comment (da1suk8) [reply x/wasm/keeper/querier_test.go:315]: Understood, thank you!

**Review by da1suk8 — APPROVED** [review]

**Review by loloicci — APPROVED** [review]

**Pull request merged**